.test-defs/RegisterSeed.yaml

```diff
@@ -0,0 +1,30 @@
+kind: TestDefinition
+metadata:
+  name: register-seed
+spec:
+  owner: gardener-oq@listserv.sap.com
+  description: Tests the registration of a seed.
+  activeDeadlineSeconds: 7200
+
+  command: [bash, -c]
+  args:
+  - >-
+    go test -timeout=0 -mod=vendor ./test/system/seed_registration
+    --v -ginkgo.v -ginkgo.progress -ginkgo.noColor
+    -kubecfg=$TM_KUBECONFIG_PATH/gardener.config
+    -seed-name=$SEED_NAME
+    -shoot-kubecfg=$TM_KUBECONFIG_PATH/shooted-seed.config
+    -secret-binding=$SECRET_BINDING
+    -shoot-name=$SHOOT_NAME
+    -shoot-namespace=$PROJECT_NAMESPACE
+    -backup-secret-provider=$BACKUP_SECRET_PROVIDER
```

Comment:
Can the backup parameters be optional for the test run. For example I want to run the test locally without specifying a backup (it is optional in the Seed spec). On the other side you probably need the backup configurable for the controlplane migration tests.

Reply:
Yep, we can make them optional in the code, but they are required for the control plane migration tests. In the future we'd like to run the test on multiple providers.